plugin/trino-hive/src/main/java/io/trino/plugin/hive/util/HiveUtil.java

```diff
@@ -184,6 +184,14 @@
     public static final String ICEBERG_TABLE_TYPE_NAME = "table_type";
     public static final String ICEBERG_TABLE_TYPE_VALUE = "iceberg";
 
+    public static final String HUDI_TABLE_TYPE_NAME = "type";
+    public static final String HUDI_TABLE_TYPE_COPY_ON_WRITE = "cow";
+    public static final String HUDI_TABLE_TYPE_MERGE_ON_READ = "mor";
+    private static final String HUDI_PARQUET_INPUT_FORMAT = "org.apache.hudi.hadoop.HoodieParquetInputFormat";
```

Comment:
Let's compare against `HoodieParquetInputFormat.class.getName()` instead of a hardcoded string

Reply:
`hudi-hadoop-mr` dependency that contains input format classes is not in the context of `trino-hive` plugin, so that those are listed as Strings. Explanation comment added: https://github.com/trinodb/trino/compare/981ad14466401fe25bf79fe951d3d9f60003b820..1b85603f5f689164d884d9a82b879ea818712eb4#diff-9185e5edfda3aecb853ca7c1dc8d11da091a035d61503deca685034430dfaabeR190